src/m365/spo/commands/site/site-list.ts

```diff
@@ -110,15 +115,53 @@ class SpoSiteListCommand extends SpoCommand {
     });
   }
 
+  /* 
+    The type property currently defaults to Teamsite. 
+    It makes more sense to default to All. Certainly after adding the 'includeOneDriveSites' option.
+    Changing this will be a breaking change. We'll remove the default the next major version.
+  */
+  private getWebTemplateId(options: Options): string {
+    if (options.webTemplate) {
+      return options.webTemplate;
+    }
+
+    if (options.includeOneDriveSites) {
+      return '';
+    }
+
+    let siteType = options.type;
+
+    if (!siteType) {
+      siteType = 'TeamSite';
```

Comment:
I think if this is empty we could just return `return 'GROUP#0'` 🤔 not needed to fill this variable with some value in order to proceed in further logic if we may stop the complexity (as simple as is) already here?

Reply:
I will do it when merging